laos-ownership-indexer/example.env

```diff
@@ -8,4 +8,8 @@ TRANSFER_MODEL=PolygonTransfer
 SCHEMA_NAME=ownership_chain_polygon_processor
 
 #Configuration
-DB_PORT=23798
\ No newline at end of file
+DB_NAME=squid

```

Comment:
can you show me a line of code where any of the following env variables is used?

DB_NAME=squid
DB_USER=postgres
DB_PASS=postgres
GQL_PORT=4351